utilities/generate_api.py

```diff
@@ -40,6 +42,33 @@
 JAVA_GAPIC="java_gapic"
 JAVA_DISCOGAPIC="java_discogapic"
 
+
+def dump_versions(config_path):
+    print("Component versions:")
+
+    print(subprocess.check_output(['artman', '--version'], stderr=subprocess.STDOUT).strip())
+
+    print("googleapis %s" % get_git_repo_version(os.path.dirname(config_path)))
+    print("google-cloud-java %s" % get_git_repo_version(os.path.dirname(__file__)))
+
+    with io.open(os.path.expanduser("~/.artman/config.yaml"), encoding='UTF-8') as config_file:
+        artman_config_data = yaml.load(config_file, Loader=yaml.Loader)
+        toolkit_path = artman_config_data['local']['toolkit']
+        print("gapic_generator  %s" % get_git_repo_version(toolkit_path))
+
+
+def get_git_repo_version(path):
```

Comment:
Move this above `dump_versions` so that the ordering in the file is consistent (lowest-level functions at the top)

Reply:
done